timer-selector.js

```diff
@@ -248,6 +248,50 @@ function getMessage(language, messageKey) {
       'error-missing': 'Hãy chọn đầy đủ các tùy chọn bắt buộc',
       'error-invalid': 'Lỗi phần mềm, vui lòng gửi báo cáo lỗi.',
     },
+    // Chinase
+    'zh': {
+      'timer-description': '此计时器将到期于：',
+      'timer-progress': '此计时器将在下列时间后过期：',
+      'timer-finished': '此计时器已过期：',
+      'timer-type': '计时器类型',
+      'timer-type-generic': '通用',
+      'timer-type-deletion': '删除',
+      'timer-type-ban': '封禁',
+      'duration': '运行时间',
+      'duration-1d': '1天',
```

Comment:
I learn something new every day. I do not know Chinese. I would have guessed this item would be "1日". I did not know Chinese used "天" for day, that is interesting.

(You do not need to edit anything. I am commenting only.)

Reply:
后来确实是修改为了“日”，因为“天”可能有一点点口语化吧（至少我觉得吧
呃嗯，不是特别清楚吧，“天”可以表示一整天吧
具体的可以看看一些百科？
Later, it was really revised to "day", because "day" may be a little colloquial (at least I think
Well, I don't know, "day" can mean a whole day
Can I have a look at some encyclopedias?

Reply:
Yes I can read more about it. It is interesting.